app/src/main/java/com/example/asiantech/playmusic/MainActivity.java

```diff
@@ -0,0 +1,453 @@
+package com.example.asiantech.playmusic;
+
+import android.app.ActivityManager;
+import android.content.BroadcastReceiver;
+import android.content.ComponentName;
+import android.content.ContentResolver;
+import android.content.Context;
+import android.content.Intent;
+import android.content.IntentFilter;
+import android.content.ServiceConnection;
+import android.database.Cursor;
+import android.net.Uri;
+import android.os.Handler;
+import android.os.IBinder;
+import android.provider.MediaStore;
+import android.support.v4.app.Fragment;
+import android.support.v4.app.FragmentActivity;
+import android.support.v4.app.FragmentTransaction;
+import android.support.v4.view.ViewPager;
+import android.util.Log;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.SeekBar;
+import android.widget.TextView;
+
+import com.astuetz.PagerSlidingTabStrip;
+import com.example.asiantech.playmusic.adapter.PagerAdapter;
+import com.example.asiantech.playmusic.fragment.PlaySongFragment_;
+import com.example.asiantech.playmusic.fragment.SongListFragment_;
+import com.example.asiantech.playmusic.model.Song;
+import com.example.asiantech.playmusic.service.MusicService;
+
+import org.androidannotations.annotations.AfterViews;
+import org.androidannotations.annotations.Click;
+import org.androidannotations.annotations.EActivity;
+import org.androidannotations.annotations.ViewById;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.Formatter;
+import java.util.Locale;
+
+import lombok.Getter;
+
+@EActivity(R.layout.activity_main)
+public class MainActivity extends FragmentActivity implements OnItemListener {
+
+    @ViewById(R.id.tabStrip)
+    PagerSlidingTabStrip mTabStrip;
+    @ViewById(R.id.viewPager)
+    ViewPager mViewPager;
+    @ViewById(R.id.imgPlay)
+    ImageView mImgPlay;
+    @ViewById(R.id.seekBar)
+    SeekBar mSeekBar;
+    @ViewById(R.id.tvCurrentTime)
+    TextView mTvCurrentTime;
+    @ViewById(R.id.tvSongTime)
+    TextView mTvSongTime;
+    @ViewById(R.id.llController)
+    LinearLayout mController;
+    @ViewById(R.id.tvSongTitle)
+    TextView mTvSongTitle;
+    @Getter
+    private OnItemListener mOnItemListener;
+    @Getter
+    private ArrayList<Song> mListSong;
+    private boolean mBound;
+    @Getter
+    private MusicService mMusicService;
+    private Intent mPlayIntent;
+    private StringBuilder mFormatBuilder;
+    private Formatter mFormatter;
+    private Handler mHandler = new Handler();
+    public static final String ACTION_STRING_ACTIVITY = "ToActivity";
+    public static final String LIST_TYPE_ALL_SONG = "AllSong";
+    public static final String LIST_TYPE_ALBUM = "Album";
+    public static final String LIST_TYPE_ARTIST = "Artist";
+    private PagerAdapter mPagerAdapter;
+    private String mMessage = "";
+
+    @AfterViews
+    void afterView() {
+        Log.d("TAG ACTIVITY", "afterView");
```

Comment:
WARNING: [AvoidDuplicateLiterals] The String literal "TAG ACTIVITY" appears 24 times in this file; the first occurrence is on line 86
http://pmd.sourceforge.net/pmd-5.2.3/pmd-java/rules/java/strings.html#AvoidDuplicateLiterals